pipert2/core/base/synchronise_routines/routines_synchroniser.py

```diff
@@ -45,34 +44,36 @@ def create_routines_graph(self):
 
         """
 
-        synchronise_graph = {}
-        synchroniser_nodes = {}
+        synchronize_graph = {}
+        synchronizer_nodes = {}
+
+        routines_with_type = infer_routines_types(self.wires.values())
 
         for wire in self.wires.values():
             for wire_destination_routine in wire.destinations:
-                if wire_destination_routine.name not in synchroniser_nodes:
-                    synchroniser_nodes[wire_destination_routine.name] = SynchroniserNode(
+                if wire_destination_routine.name not in synchronizer_nodes:
+                    synchronizer_nodes[wire_destination_routine.name] = SynchroniserNode(
```

Comment:
If you are already changing the spelling from synchroniser to synchronizer, correct every instance.